### PR TITLE
export() returns list of file names created

### PR DESCRIPTION
**Pull request by drnikolaev**
Signed-off-by: Sergei Nikolaev <snikolaev@nvidia.com>

### CODE REVIEW

File: nemo/collections/asr/models/classification_models.py [hunk 1]
```diff
@@ -402,6 +404,7 @@ def export(
 
         output_model = attach_onnx_to_onnx(encoder_onnx, decoder_onnx, "EDC")
         onnx.save(output_model, output)
+        return [output, output1, output2]
```
Comment (tkornuta-nvidia) [inline nemo/collections/asr/models/classification_models.py:407]: awesome! 👏

**Review by tkornuta-nvidia — APPROVED** [review]
Hey @drnikolaev this really looks great!

Note in EFF you can add any number of file properties, but the one I find very useful is description. Example:
```
INFO: trained-model.tlt [RESTORING]
==================================================
metadata:
  format_version: 1
  encryption: false
  obj_cls: nemo.collections.nlp.models.text_classification.text_classification_model.TextClassificationModel
  origin: NeMo
  nemo_archive_version: 1
  runtime: PyTorch
--------------------------------------------------
files:
  model_config.yaml: File containing model configuration
  model_weights.ckpt: File containing model weights
==================================================
```

Hence I have a tiny request - could you return, aside of list of filenames, also list of descriptions, one per file? Not need to be sophisticated, but e.g. "Encoder exported to ONNX" (e.g. template: module {} exported to ONNX)

Still, I don't find it crucial, hence approving.

**Review by okuchaiev — APPROVED** [review]
This looks fine to me. But I am wondering where it will be used?

**Comment by tkornuta-nvidia** [comment]
> This looks fine to me. But I am wondering where it will be used?

In EFF: ONNXCookbook :)

**Review by tkornuta-nvidia — APPROVED** [review]
Great, thanks @drnikolaev !! :)

**Pull request merged**